### PR TITLE
 For #25212: Add tabs prioritization 

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt
Comment (Amejia481) [inline app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:386]: @Mugurell 👋🏽 
I added this to address the AC breaking change, I took it from https://github.com/mozilla-mobile/fenix/pull/25290. I'm not sure how close it's from landing as there some still discussions on the PR. Is this approach OK ?  I noticed `getFakePocketStories` is only being used in the preview.

Comment (Mugurell) [reply app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:386]: Looks good.
Was waiting for the AC bump to merge.
Seems like we can merge this and I'll land immediately after.
Thank you for the new AC version and a swifter merge process! 😊

**Review by Mugurell — APPROVED** [review]

File: app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt
Comment (Mugurell) [reply app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:386]: Seems like some unit tests would also need to be updated.
As discussed I'll borrow the AC version bump and land first to address all breaking changes.

**Pull request merged**
